File: CS/MySolution.Module/BusinessObjects/Employee.cs
using DevExpress.ExpressApp.DC;
using DevExpress.Persistent.Base;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MySolution.Module {
    [DefaultClassOptions, ImageName("BO_Employee")]
    [MetadataType(typeof(EmployeeMetadata))]
    public partial class Employee {
    }
    public class EmployeeMetadata {
        [Browsable(false)]
        public Int32 Id { get; set; }
    }

}

File: VB/MySolution.Module/BusinessObjects/Employee.vb
Imports DevExpress.ExpressApp.DC
Imports DevExpress.Persistent.Base
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.ComponentModel.DataAnnotations
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace MySolution.Module
    <DefaultClassOptions, ImageName("BO_Employee"), MetadataType(GetType(EmployeeMetadata))> _
    Partial Public Class Employee
    End Class
    Public Class EmployeeMetadata
        <Browsable(False)> _
        Public Property Id() As Int32
    End Class

End Namespace
